specs/core/1_shard-data-chains.md

```diff
@@ -300,76 +302,52 @@ def is_valid_shard_block(beacon_blocks: Sequence[BeaconBlock],
             return True
 
     # Check slot number
-    assert candidate.slot >= PHASE_1_FORK_SLOT
-
-    # Check shard number
-    assert candidate.shard <= SHARD_COUNT
+    assert candidate.core.slot >= PHASE_1_FORK_SLOT
 
     # Check beacon block
-    beacon_block = beacon_blocks[candidate.slot]
-    assert candidate.beacon_block_root == signing_root(beacon_block)
-    assert beacon_block.slot <= candidate.slot
+    beacon_block = beacon_blocks[candidate.core.slot - candidate.core.slot % (SHARD_SLOTS_PER_BEACON_SLOT * SLOTS_PER_EPOCH)]
+    assert candidate.core.beacon_block_root == signing_root(beacon_block)
+    assert beacon_block.slot <= candidate.core.slot
 
     # Check state root
-    assert candidate.state_root == Hash()  # [to be removed in phase 2]
+    assert candidate.core.state_root == Hash()  # [to be removed in phase 2]
```

Comment:
`state_root` represents the shard system state which will be around in phase 1.
We should remove this line

Reply:
Agree! But I'm deliberately leaving this as is so that we can add the system state in a separate PR.